src/cozmo/conn.py

```diff
@@ -356,7 +356,9 @@ def _handle_ui_device_connected(self, msg):
         self._is_ui_connected = True
         self.dispatch_event(EvtConnected, conn=self)
         self.anim_names.refresh()
-        logger.info("UI device connected")
+        logger.info('App connection established.  sdk_version=%s  '
+                'cozmoclad_version=%s app_build_version=%s',
```

Comment:
spacing/formatting seems a bit random, and will wrap with a terminal narrower than 160 chars
`2016-11-02 17:43:40,510 cozmo.general INFO     App connection established.  sdk_version=0.8.1.dev0  cozmoclad_version=1.0.2 app_build_version=00001.00000.00002`